nltk/parse/dependencygraph.py

```diff
@@ -334,6 +334,9 @@ def extract_10_cells(cells):
             # Make sure that he fake root node has labeled dependencies.
             if (cell_number == 3) and (head == 0):
                 rel = 'ROOT'
+            # Stanford parser doesn't capitalize ROOT
```

Comment:
That would be better! But, it looks like only malt.py got changed and the necessary changes to dependencygraph.py didn't make it in to #1049.